jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildDockerSteps.java

```diff
@@ -54,12 +56,35 @@ public BuildDockerSteps(
     this.buildConfiguration = buildConfiguration;
     this.sourceFilesConfiguration = sourceFilesConfiguration;
     this.cachesInitializer = cachesInitializer;
+    startupMessage =
```

Comment:
Can make this a `private static final` format and call `String.format` with the target image reference.